src/pages/messages/components/DoiRequestMessagesColumn.tsx

```diff
@@ -2,30 +2,38 @@ import { Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { ExpandedTicket, Ticket } from '../../../types/publication_types/ticket.types';
 import { toDateString } from '../../../utils/date-helpers';
+import { LastMessageBox } from './LastMessageBox';
 import { StyledMessagesContainer, StyledStatusMessageBox } from './PublishingRequestMessagesColumn';
 
 interface DoiRequestMessagesColumnProps {
   ticket: ExpandedTicket | Ticket;
+  showLastMessage?: boolean;
 }
 
-export const DoiRequestMessagesColumn = ({ ticket }: DoiRequestMessagesColumnProps) => {
+export const DoiRequestMessagesColumn = ({ ticket, showLastMessage }: DoiRequestMessagesColumnProps) => {
   const { t } = useTranslation();
 
   return (
     <StyledMessagesContainer>
       {ticket.status === 'New' || ticket.status === 'Pending' ? (
-        <StyledStatusMessageBox sx={{ bgcolor: 'secondary.dark' }}>
-          <Typography>{t('my_page.messages.doi_pending')}</Typography>
-        </StyledStatusMessageBox>
+        <>
+          {showLastMessage && <LastMessageBox ticket={ticket as ExpandedTicket} />}
+          <StyledStatusMessageBox sx={{ bgcolor: 'secondary.dark' }}>
+            <Typography>{t('my_page.messages.doi_pending')}</Typography>
+          </StyledStatusMessageBox>
+        </>
       ) : (
-        <StyledStatusMessageBox sx={{ bgcolor: 'doiRequest.main' }}>
-          {ticket.status === 'Completed' ? (
-            <Typography>{t('my_page.messages.doi_completed')}</Typography>
-          ) : ticket.status === 'Closed' ? (
-            <Typography>{t('my_page.messages.doi_closed')}</Typography>
-          ) : null}
-          {ticket.modifiedDate && <Typography>{toDateString(ticket.modifiedDate)}</Typography>}
-        </StyledStatusMessageBox>
+        <>
+          {showLastMessage && <LastMessageBox ticket={ticket as ExpandedTicket} />}
```

Comment:
her er en annen litt skummel ting, som er at `showLastMessage` betyr at man er på oppgave-siden hvor man søker på tickets, som betyr at dette faktisk er `ExpandedTicket`, men det er jo på ingen måte åpenbart, og er egentlig bare en snarvei hvor vi presser typene til å bli riktig fordi vi "vet" at det er sånn, i stedet for å bare lene oss på robust kode. Dette kommer egentlig som en følge av den forrige [kommentaren](https://github.com/BIBSYSDEV/NVA-Frontend/pull/6174/files#r1741011506) min

Reply:
Ja det høres jo ikke egentlig bra ut. Er det sånn at det er gjort så mange småendringer her og der at det begynner å tre frem et behov for en større refaktorering? Det går jo an å gjøre det i en ny PR?

Reply:
Ja, syns det lukter litt dårlig av dette selv... Men lurer på om det lukter akkurat lite nok til at jeg gjør noe med det nå 🤔 
Men vi/jeg kan hvertfall prøve å bli mer bevisst på sånne ting fremover, da feilen her på en måte ble introdusert tidligere, ved at den ble gjenbrukt uten å egentlig skulle gjøre akkurat det samme. Tror fort der er en sånn ting som alltdi vil være både-og da :/